pyvista/utilities/features.py

```diff
@@ -176,3 +178,72 @@ def transform_vectors_sph_to_cart(theta, phi, r, u, v, w):
     w_t = np.cos(ph) * w - np.sin(ph) * v
 
     return u_t, v_t, w_t
+
+
+def merge(
+        datasets,
+        merge_points=True,
+        main_has_priority=True,
+        progress_bar=False
```

Comment:
Just future-proofing
```suggestion
        progress_bar=False,
```